CHANGELOG.md

```diff
@@ -6,6 +6,13 @@
 
 - Add Amazon `BedrockEmbedding` (#8550)
 
+## [0.8.56] - 2023-10-30
+
+### Bug Fixes / Nits
+
+- In Knowledge Graph Index with hybrid retriever_mode, 
+  - return the nodes found by keyword search when 'No Relationship found' from embedding search.
```

Comment:
Please move this up to under ## Unreleased section. I also think you can omit the words "from embedding search" in your comment for added brevity.

Reply:
Updated.